### PR TITLE
Feature/css

### PR DESCRIPTION
**Pull request by lulup52**
We create the css file and work on it to get our fist visual style.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,34 +1,45 @@
 import React from 'react';
-import SearchBar from './components/SearchBar';
-import './App.css';
-/*import CocktailList from './components/CocktailList';*/
 import Axios from 'axios';
+import SearchBar from './components/SearchBar';
+import './components/makeYourCocktail.css';
+/* import CocktailList from './components/CocktailList'; */
```
Comment (rocambille) [inline src/App.jsx]: dead code: remove it :)

File: src/App.jsx [hunk 2]
```diff
@@ -1,34 +1,45 @@
...
 render() {
   return (
-    <div className="App">
-      <SearchBar setKeywords={this.setKeywords} onSearch={this.search} />
-  {this.state.cocktails.map((cocktail) => <li>{cocktail}</li>)}
+    <div>
+      <head>
+        <style>@import url(`https://fonts.googleapis.com/css2?family=Kalam:wght@400;700&display=swap`);</style>
+      </head>
+      <body>
```
Comment (rocambille) [inline src/App.jsx]: body can't be a child of a div. move your style into public/index.html

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (lulup52) [reply src/App.jsx]: witch code is dead ? just the line of comment ?

**Review by lulup52 — COMMENTED** [review]
problem solved

**Pull request merged**